### ISSUE TITLE
Update LIRMM repository

### ISSUE THREAD

**Issue by CK-Explorer** [BenjaminNavarro/ellipsoid-fit#9]
Hi Benjamin, how can I update the project in LIRMM repo?

**Comment by BenjaminNavarro** [comment]
It's up to @passama to do it now since I'm not at LIRMM anymore

**Comment by passama** [comment]
Done. Be warn that I am not a member of this project so I cannot push back any modification from LIRMM's gitlab. Automatic mirrorring to github also no more work, probably due to new github security policy.

**Comment by CK-Explorer** [comment]
Noted, it's too bad that mirroring is not working. Thanks Benjamin and passama in looking into this issue, and update the repo.

**Comment by passama** [comment]
Basically if @BenjaminNavarro register me as member of this project I can try to solve this issue ;-)

**Comment by passama** [comment]
thanks Benjamin !

**Comment by BenjaminNavarro** [comment]
Done. We can also move the project to LIRMM's Github account if that's easier for you Robin

**Comment by passama** [comment]
Yes why not. This could also help you to avoid unwanted notifications ;-)

**Comment by BenjaminNavarro** [comment]
Project moved. I should check if I have some other repos that should be moved as well

**Comment by passama** [comment]
Thanks a lot !
